demo/app/script.ts

```diff
@@ -27,9 +27,11 @@ space.on('membersUpdate', (members) => {
 
 /** Avoids issues with top-level await: an alternative fix is to change build target to es */
 (async () => {
+  space.once('membersUpdate', () => {
+    renderSelfAvatar(selfName);
+    renderFeatureDisplay(space);
+    const initialMembers = space.getMembers();
+    renderAvatars(initialMembers.filter(memberIsNotSelf));
+  });
   await space.enter({ name: selfName });
```

Comment:
Should we not just put:
```
  space.once('membersUpdate', () => {
    renderSelfAvatar(selfName);
    renderFeatureDisplay(space);
    const initialMembers = space.getMembers();
    renderAvatars(initialMembers.filter(memberIsNotSelf));
  });
```

outside of the `async` block? then I think we can do:

```
const init = async () => {
  await space.enter({ name: selfName });
}

init();
```